ruthyn-example.c

```diff
@@ -0,0 +1,46 @@
+#include <stdio.h>
+
+/* ruthyn-example.c
+ *
+ * Tests out the the Unicode range between U+0400 and U+047F, which
+ * includes all the ruthyn-hd44780 characters. The string literals in
+ * this file are encoded in UTF-8 for easy development. When compiled
+ * correctly, this program actually emits the Ruthyn single byte
+ * encoding, not UTF-8.
+ *
+ * Compilation:
+ *	GCONV_PATH=../gconv gcc  -fexec-charset=ruthyn  ruthyn-example.c
+ *
+ * Required during compilation:
+ *	../gconv/gconv-modules				# Gconv catalog
+ *	../gconv/ruthyn-hd44780.so			# Gconv module
+ *
+ * Once compiled, the executable does _not_ require the gconv files.
```

Comment:
Again, right, this is true. But this must be obvious for the reader. If not then we made a mistake in the readme...

Reply:
Again, will revert.